projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -759,16 +772,151 @@ function wpcom_add_new_page_check( $post_id, $post ) {
 }
 add_action( 'wp_insert_post', 'wpcom_add_new_page_check', 10, 3 );
 
+/**
+ * Determine if the site has an 'About' page.
+ * We do this by loading the `en` annotation for the site and theme.
+ * We then check if there is a post with the title 'About'. If there is, we get the `hs_old_id` and check to make sure a corresponding post exists in the user's locale.
+ *
+ * @return int|null The page ID of the 'About' page if it exists, null otherwise.
+ */
+function wpcom_get_site_about_page_id() {
+	// First, attempt to get the page ID from the cache.
+	$blog_id                 = get_current_blog_id();
+	$about_page_id_cache_key = "wpcom_get_site_about_page_id_{$blog_id}";
+	$about_page_id           = wp_cache_get( $about_page_id_cache_key, 'wpcom' );
+	if ( false !== $about_page_id ) {
+		return $about_page_id;
+	}
+
+	// If we don't have a cached page id yet, go through the process of looking it up.
+	$annotation = wpcom_get_theme_annotation( get_stylesheet() );
+
+	// Return null if there is no annotation or the annotation doesn't have any content.
+	if ( ! $annotation || ! isset( $annotation['content'] ) || ! is_array( $annotation['content'] ) ) {
+		return null;
+	}
+
+	// Use the annotation to build up a list of 'About' pages.
+	$headstart_about_pages = array_filter(
+		$annotation['content'],
+		function ( $page ) {
+			if ( 'page' !== $page['post_type'] ) {
+				return false;
+			}
+
+			if ( 'about' === $page['post_name'] || false !== strpos( $page['post_title'], 'About' ) ) {
+				return true;
+			}
+		}
+	);
+
+	// Return null if there are no 'About' pages.
+	if ( empty( $headstart_about_pages ) ) {
+		return null;
+	}
+
+	// Get the hs_old_ids for the 'About' pages. We'll use these to find any published pages with the same hs_old_id.
+	$headstart_about_page_hs_old_ids = array_map(
+		function ( $about_page ) {
+			return $about_page['hs_old_id'];
+		},
+		$headstart_about_pages
+	);
+
+	// Return null if there aren't any hs_old_ids to check.
+	if ( empty( $headstart_about_page_hs_old_ids ) ) {
+		return null;
+	}
+
+	$filters     = array(
```

Comment:
Based on the docs [here](https://developer.wordpress.org/reference/classes/WP_Query/parse_query/), we may also want to add `'fields' => 'ids'` below so we only get the post IDs returned.